ui/src/components/TableRow.js

```diff
@@ -36,12 +35,11 @@ export const UnknownIcon = styled.i`
 `;
 
 const TableRow = (props) => {
-  const { row, style, onClickRow, isSelected } = props;
+  const { row, style, isSelected } = props;
 
   return (
     <TableRowStyle
       {...row.getRowProps({
-        onClick: () => onClickRow(row),
```

Comment:
Would be cool to remove the highlight of the row, since we won't be able to select the row anymore. 
@Cuervino Can you confirm that please?

![image](https://user-images.githubusercontent.com/18453133/111185317-223c9280-85b2-11eb-8943-61c4051e1373.png)

Reply:
Yes, indeed, here selecting the line doesn't have any added value. Moreover, the table in the "Pods" tab already has this "non-selectable lines" logic.